koku/api/provider/view.py

```diff
@@ -271,7 +271,7 @@ def destroy(self, request, *args, **kwargs):
             @apiVersion 1.0.0
             @apiDescription Delete a provider.
 
-            @apiHeader {String} token Service Admin authorization token.
+            @apiHeader {String} token User authorization token.
```

Comment:
If you want to be specific here its either the *Created By Use* or the *Customer Owner*.

Reply:
I like that even better. Updating the doc